Containerfile

```diff
@@ -0,0 +1,5 @@
+FROM golang:1.19
+WORKDIR  /go/src/github.com/janus-idp/webterminal-proxy
+COPY . .
+RUN go build .
```

Comment:
There is no need for all build artifacts, dependencies, source code to be present in the resulting image. The resulting binary is enough. Please inspire in the dockerfile you've linked and use a multistage build here.

https://github.com/openshift/oauth-proxy/blob/03e5b13b8b7087dd70abfd70efb4c5b92f800a4f/Dockerfile#L6-L8

You should be able to use `FROM scratch` as the resulting image base. If not (like if you run into cert issues within openshift or something, you may need to change this to something like `ubi9/minimal` or `ubi9/micro`)

https://chemidy.medium.com/create-the-smallest-and-secured-golang-docker-image-based-on-scratch-4752223b7324

Reply:
I ran into issues with dynamic libraries when using `scratch` I think it should be possible to resolve the problem but using `ubi9/micro` creates 34 MB image vs 7.8 MB `scratch` image considering that I would have to add additional libraries to that image I think the resulting difference would be smaller, so for now I have `ubi9/micro` as a base image there.